.github/workflows/python-app.yml

```diff
@@ -11,18 +11,19 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```

Comment:
This lib _requires_ >= 3.9, because of [`exit_on_error` params in argparse](https://docs.python.org/3/library/argparse.html#exit-on-error).